resources/compass/charts/kyma-environment-broker/templates/tests/e2e-provisioning-gcp.yaml

```diff
@@ -69,6 +69,16 @@ spec:
               value: {{ .Values.gardener.kubeconfigPath }}
             - name: APP_TEST_AZURE_EVENT_HUBS_ENABLED
               value: "true"
+            - name: APP_UPGRADE_TEST
+              value: "false"
+            - name: APP_MANAGED_RUNTIME_COMPONENTS_YAML_PATH
+              value: /config/additionalRuntimeComponents.yaml
+            - name: APP_UPGRADE_TIMEOUT
+              value: "100m"
```

Comment:
please put that value into `values.yaml` file under `e2e.upgrade.timeout`. I have a PR in which I do the same with provisioning and deprovisioning timeouts.